cloud-formation/python/geonames-lambda.py

```diff
@@ -0,0 +1,33 @@
+"""
+Function: Geonames-lambda
```

Comment:
The function not be called by a API name because it is one function to get all params and parse.